reconcile/test/cna/test_client.py

```diff
@@ -0,0 +1,45 @@
+from reconcile.cna.client import CNAClient
+
+
+def test_client_asset_type_metadata_init():
+    pass
+
+
+def test_client_list_assets_for_creator(mocker):
+    creator = "creator"
+    listed_assets = [
+        {
+            "asset_type": "null",
+            "id": "123",
+            "href": "url/123",
+            "status": "Running",
+            "creator": {"username": creator},
+        },
+        {
+            "asset_type": "null",
+            "id": "456",
+            "href": "url/456",
+            "status": "Running",
+            "creator": {},
+        },
+        {
+            "asset_type": "null",
+            "id": "789",
+            "href": "url/789",
+            "status": "Running",
+        },
+        {
+            "asset_type": "null",
+            "id": "000",
+            "href": "url/000",
+            "status": "Running",
+            "creator": {"username": "another_user"},
+        },
+    ]
+
+    mocker.patch.object(CNAClient, "list_assets", return_value=listed_assets)
+    cna_client = CNAClient(None)  # type: ignore
```

Comment:
nit: would `create_autospec(OCMBaseClient)` work here?

Reply:
i autospeced the patch only 

`mocker.patch.object(CNAClient, "list_assets", return_value=listed_assets, autospec=True)`

i verified that this is sufficient to ensure that `list_assets_for_creator` calls `listed_assets` correctly.

is this ok with you or do you prefer `create_autospec`?

Reply:
I was thinking more of the fact that we're passing `None` in and need to ignore a type error? If we used `create_autospec` as a Mock factory, then I don't think you'd need to do this?